converter/biosamples.py

```diff
@@ -59,10 +59,12 @@ def __add_project_attribute(sample):
         )
 
     @staticmethod
-    def __convert_datetime(datetime_str: str) -> datetime:
-        if datetime_str:
+    def __convert_datetime(date_time: any) -> datetime:
+        if isinstance(date_time, str):
```

Comment:
what happens if date_time is not a string? shouldn't we raise an exception?

Reply:
Done